index.html

```diff
@@ -10,20 +10,50 @@
 
     <h1>Helen Abera</h1>
 
+    <header>
+        <nav class="nav-bar">
+          <ul class="nav-links">
+            <li><a href="#">About</a></li>
+            <li><a href="#">Experience</a></li>
+            <li><a href="#">Skills</a></li>
+            <li><a href="#">Projects</a></li>
+            <li><a href="#">Connect</a></li>
```

Comment:
I think here at the end of the lesson it's asking you to add the actual ids here in the href so that the nav will be functional